src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -558,6 +562,7 @@ const CheckoutLoadingManager: React.FC<{
       [
         minimumLoadingPassed,
         orderValidated,
+        isExpressCheckoutLoaded,
         isPartnerOfferLoadingComplete,
         isLoading,
```

Comment:
I think that removing this [causes the `state.isLoading` check to not work for setting the express checkout payment methods list](https://github.com/artsy/force/blob/4d263e8a65a60e01ed6b4e125a68ec8719324683/src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx#L140-L145). As a result it never gets set, causing the page to render an empty express checkout section if no express checkout methods are available. Adding this back should fix it but it's possible we can just skip the `state.isLoading` check.
<img width="1497" height="944" alt="image" src="https://github.com/user-attachments/assets/a8acbf13-9409-40eb-be69-b13c838dc922" />

Reply:
Question: Could we put the stripe provider at a higher level (around the app/route) so we don't need to ever render the actual UI?